lib/rubocop/cop/lint/duplicate_set_element.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # Checks for duplicate elements in Set.
+      #
+      # @example
+      #
+      #   # bad
+      #   Set[:foo, :bar, :foo]
```

Comment:
should this cop also detect `[:foo, :bar, :foo].to_set`?

I know a few codebases where this is preferred to `Set[...]`

Reply:
This case seems to be reasonable. I've updated this PR.